pkg/apis/minio.min.io/v1/names.go

```diff
@@ -66,8 +66,10 @@ func (t *Tenant) MinIOCIServiceName() string {
 }
 
 // MinIOCSRName returns the name of CSR that is generated if AutoTLS is enabled
+// Namespace adds uniqueness to the CSR name (single MinIO tenant per namsepace)
+// since CSR is not a namespaced resource
 func (t *Tenant) MinIOCSRName() string {
-	return t.Name + CSRNameSuffix
+	return t.Name + t.Namespace + CSRNameSuffix
```

Comment:
add separation between name and namespace?

Reply:
you mean like a - or . ?